ChopChop/Models/Cooking/SessionRecipeStepNode.swift

```diff
@@ -1,21 +1,24 @@
 import SwiftUI
 
-class SessionRecipeStepNode: Node, ObservableObject {
+final class SessionRecipeStepNode: Node, ObservableObject {
+    let id = UUID()
     var label: SessionRecipeStep
-    @Published var isCompletable: Bool = false
-    @Published var isCompleted: Bool = false
+    var position: CGPoint?
```

Comment:
I can import Combine and CoreGraphics instead of SwiftUI, it should be the same thing. As for CoreGraphics, since we're only using the data structures, it seems reasonable to me (unless we want to create our own graphic data structures).